asu/build.py

```diff
@@ -300,6 +300,42 @@ def download_file(filename: str, dest: str = None):
             cache_workdir / ".config",
         )
 
+    rootfssize = None
+    if req.get("rootfssize") and isinstance(req.get("rootfssize"), int):
+        if 1 < req.get("rootfssize") <= req["max_custom_rootfs_size_mb"]:
+            rootfssize = req.get("rootfssize")
+        elif req.get("rootfssize") > req["max_custom_rootfs_size_mb"]:
+            rootfssize = req["max_custom_rootfs_size_mb"]
```

Comment:
Use openapi max and minimum to verify the data instead of checking it here afterwards.